includes/wf_crm_webform_postprocess.inc

```diff
@@ -1472,6 +1472,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
   * Process the submission into the details of the activity.
   */
  private function formatSubmissionDetails(&$params, $activity_number) {
+   /*
```

Comment:
Commented this out. We can keep, just need to generate the correct route. But now it will submit.